### ISSUE TITLE
데이콘 28등 사용 모델 및 파라미터 정리 ( f1_score @0.991 / 리더보드 : 우르르롹끼 )

### ISSUE THREAD

**Issue by 5JM** [Sangmyung-University-PrLab/AnomalyDetection#2]
**사용 모델** : 
  ByobNet -> timm.create_model("regnetz_e8", pretrained = true, num_classes = 88)

**loss**  : 
  CrossEntropyLoss

**optimizer** : 
  Adam

**lr_scheduler** : 
  torch.optim.lr_scheduler.ReduceLROnPlateau

**data agumentation** : 

               ColorJitter, RandomBrightnessContrast, GaussNoise,

               _oneOf_( HorizontalFlip, RandomRotate90, VerticalFlip ),

               _oneOf_( GridDistortion, ShiftScaleRotate, ElasticTransform),

               Normalize(mean=[0.485, 0.456, 0.406], std = [0.229, 0.224, 0.225])
